content/pt-br/docs/reference/glossary/cluster-operator.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Operador de Cluster
+id: cluster-operator
+date: 2018-04-12
+full_link: 
+short_description: >
+  Uma pessoa que configura, controla e monitora clusters.
+
+aka: 
+tags:
+- user-type
+---
+  Uma pessoa que configura, controla e monitora clusters.
+
+<!--more--> 
+
+Sua principal responsabilidade é manter um cluster funcionando, o que pode envolver atividades periódicas de manutenção ou atualizações.<br>
+
+{{< note >}}
+Os operadores de cluster são diferentes do [padrão Operador](https://www.redhat.com/pt-br/technologies/cloud-computing/openshift/what-are-openshift-operators) que estende a API Kubernetes.
```

Comment:
Thanks for picking up the translation in Portuguese for the original link! 🎉

Reply:
+1